src/person/serializers.py

```diff
@@ -33,7 +33,7 @@ class UserSerializer(serializers.ModelSerializer):
     prompts = SerializerMethodField("get_prompts")
     rides = SerializerMethodField("get_rides")
 
-    def get_prompts(self, user):
+    def get_prompts(self, user: User):
```

Comment:
Are other inputs typed as well

Reply:
I had some warnings when creating the schema.yml file, so I thought typing some of the inputs would make them go away (and it did for some but not for others). By the way, the command to create the schema file is 
`python manage.py spectacular > schema.yml`